recipes/python-pathlib2/meta.yaml

```diff
@@ -0,0 +1,36 @@
+{% set version = "2.1.0" %}
+
+package:
+  name: python-pathlib2
+  version: {{ version }}
+
+source:
+  fn: pathlib2-{{ version }}.tar.gz
+  url: https://github.com/mcmtroffaes/pathlib2/archive/{{ version }}.tar.gz
+  md5: 5ef144f5050f17962e9cef35c0e93ed7
+
+build:
+  number: 0
+  script: python setup.py install --single-version-externally-managed --record=record.txt
+
+requirements:
+  build:
+    - python
+    - setuptools
+  run:
+    - python
+    - six
+
+test:
+  imports:
+    - pathlib2
+
+about:
+  home: https://github.com/mcmtroffaes/pathlib2
+  license: MIT
+  summary: "Fork of pathlib aiming to support the full stdlib Python API."
+
+extra:
+  recipe-maintainers:
+    - pelson
+    - takluyver
```

Comment:
In the specific case, that's fine.

However, more generally, I'm concerned that we're replicating the Debian maintainership model, which I think is fundamentally broken for packaging a large diverse ecosystem of software components. Specifically, there are two related issues I have with it:

- If package maintainers are separate from upstream maintainers (as here - I'm not involved at all in pathlib2 development), I think things inevitably end up languishing because of a lack of interest. I'm unlikely to notice if a new version of pathlib2 is released, and even if I do, will I be motivated to update the conda recipe?
- If packages are maintained by specific named individuals rather than as a collective team pool, it means there's a low bus factor on each individual package, and it can lead to unwanted fragmentation, if different people pick different ways of packaging things.

I wonder if we could develop a more automated approach for straightforward packages, that wouldn't require human intervention on each new release. E.g. maybe convert wheels into conda packages? I have been working on a tool called [flonda](https://github.com/takluyver/flonda) that can build conda packages directly from the metadata my packaging tool [flit](http://flit.readthedocs.org/en/latest/) uses. That only works on packages that have chosen to use flit, but a similar technique could be used to make conda packages from wheels.